### DISCUSSION TITLE
训练的后的预期结果很差呀

### DISCUSSION THREAD

**Discussion by DemoMoon** [PaddlePaddle/PaddleSpeech#599] [Q&A]
系统：18.04.1-Ubuntu
paddlepaddle：Version: 2.0.2
DeepSpeech：develop
Python：3.8.6

/data/DeepSpeech/tools/venv/lib/python3.8/distutils/__init__.py:4: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/pkg_resources/_vendor/pyparsing.py:943: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  collections.MutableMapping.register(ParseResults)
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
-----------  Configuration Arguments -----------
checkpoint_path: ckpt-baseline/checkpoints/step-0
config: conf/deepspeech2.yaml
device: cpu
dump_config: None
export_path: None
nprocs: 1
opts: []
output: None
------------------------------------------------
data:
  augmentation_config: conf/augmentation.config
  batch_size: 64
  dev_manifest: data/manifest_zh.dev
  keep_transcription_text: False
  max_duration: 27.0
  max_freq: None
  mean_std_filepath: data/mean_std_zh.npz
  min_duration: 0.0
  n_fft: None
  num_workers: 0
  random_seed: 0
  shuffle_method: batch_shuffle
  sortagrad: True
  specgram_type: linear
  stride_ms: 10.0
  target_dB: -20
  target_sample_rate: 16000
  test_manifest: data/manifest_zh.test
  train_manifest: data/manifest_zh.train
  use_dB_normalization: True
  vocab_filepath: data/vocab_zh.txt
  window_ms: 20.0
decoding:
  alpha: 2.5
  batch_size: 128
  beam_size: 300
  beta: 5.0
  cutoff_prob: 0.99
  cutoff_top_n: 40
  decoding_method: ctc_beam_search
  error_rate_type: cer
  lang_model_path: data/lm/zh_giga.no_cna_cmn.prune01244.klm
  num_proc_bsearch: 10
model:
  num_conv_layers: 2
  num_rnn_layers: 3
  rnn_layer_size: 2048
  share_rnn_weights: False
  use_gru: True
training:
  global_grad_clip: 5.0
  lr: 0.002
  lr_decay: 0.83
  n_epoch: 5
  weight_decay: 1e-06
2021-04-21 02:50:21,344 - INFO - Setup test Dataloader!
2021-04-21 02:50:23,853 - INFO - Setup model!
-----------------ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:50:23,853 - INFO - [checkpoint] Rank 0: loaded model from ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:50:33,831 - INFO - Test Total Examples: 4
2021-04-21 02:50:33,855 - INFO - begin to initialize the external scorer for decoding
2021-04-21 02:50:33,944 - INFO - language model: is_character_based = 1, max_order = 5, dict_size = 0
2021-04-21 02:50:33,944 - INFO - end initializing scorer
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:77: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  return (isinstance(seq, collections.Sequence) and
2021-04-21 02:50:45,719 - INFO - 
Target Transcription: 猫
Output Transcription: 广州二手房市场二手房市场二手房市场二手房市场
2021-04-21 02:50:45,719 - INFO - Current error rate [cer] = 22.000000
2021-04-21 02:50:45,727 - INFO - 
Target Transcription: 由改善型换房人士承购
Output Transcription: 广州二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场
2021-04-21 02:50:45,735 - INFO - Current error rate [cer] = 11.600000
2021-04-21 02:50:45,746 - INFO - 
Target Transcription: 七月广州二手住宅市场当中
Output Transcription: 广州二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场
2021-04-21 02:50:45,758 - INFO - Current error rate [cer] = 11.500000
2021-04-21 02:50:45,777 - INFO - 
Target Transcription: 六月仍有百分之七单价在三万元的房源
Output Transcription: 广州二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场二手房市场
2021-04-21 02:50:45,796 - INFO - Current error rate [cer] = 9.764706
2021-04-21 02:50:45,796 - INFO - Error rate [cer] (4/?) = 11.050000
2021-04-21 02:50:45,796 - INFO - Test: epoch: 0, step: 0, , Final error rate [cer] (4/4) = 11.050000

**Comment by DemoMoon** [comment]
(venv) root@ip-10-0-1-86:xm_test# cat data/vocab_zh.txt 
<unk>
七
月
房
猫
广
州
二
手
住
宅
市
场
当
中
六
仍
有
百
分
之
单
价
在
三
万
元
的
源
由
改
善
型
换
人
士
承
购
(venv) root@ip-10-0-1-86:xm_test# cat data/manifest_zh.dev
{"audio_filepath": "/data/DeepSpeech/examples/xm_test/dataset/mao.flac", "duration": 0.72, "text": "猫"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0152.wav", "duration": 4.593, "text": "七月广州二手住宅市场当中"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0153.wav", "duration": 5.433, "text": "六月仍有百分之七单价在三万元的房源"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0154.wav", "duration": 3.804, "text": "由改善型换房人士承购"}
(venv) root@ip-10-0-1-86:xm_test# ll data/
total 44
drwxr-xr-x  3 root root  271 Apr 21 02:46 ./
drwxr-xr-x 11 root root  311 Apr 21 02:18 ../
drwxr-xr-x  2 root root   90 Mar 31 09:28 lm/
-rw-r--r--  1 root root  110 Mar 29 06:53 manifest_en.dev
-rw-r--r--  1 root root  110 Mar 29 06:54 manifest_en.test
-rw-r--r--  1 root root  110 Mar 29 06:54 manifest_en.train
-rw-r--r--  1 root root  724 Apr 21 02:46 manifest_zh.dev
-rw-r--r--  1 root root  106 Apr 21 02:46 manifest_zh.dev.1
-rw-r--r--  1 root root  724 Apr 21 02:47 manifest_zh.test
-rw-r--r--  1 root root  724 Apr 21 02:47 manifest_zh.train
-rw-r--r--  1 root root 3076 Mar 29 06:54 mean_std_en.npz
-rw-r--r--  1 root root 3076 Apr 21 02:47 mean_std_zh.npz
-rw-r--r--  1 root root   12 Mar 29 06:54 vocab_en.txt
-rw-r--r--  1 root root  154 Apr 21 02:47 vocab_zh.txt
(venv) root@ip-10-0-1-86:xm_test# cat data/manifest_zh.test 
{"audio_filepath": "/data/DeepSpeech/examples/xm_test/dataset/mao.flac", "duration": 0.72, "text": "猫"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0152.wav", "duration": 4.593, "text": "七月广州二手住宅市场当中"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0153.wav", "duration": 5.433, "text": "六月仍有百分之七单价在三万元的房源"}
{"audio_filepath": "/data/DeepSpeech/examples/aishell/../..//examples/dataset/aishell/data_aishell/wav/dev/S0724/BAC009S0724W0154.wav", "duration": 3.804, "text": "由改善型换房人士承购"}
(venv) root@ip-10-0-1-86:xm_test#

**Comment by DemoMoon** [comment]
/data/DeepSpeech/tools/venv/lib/python3.8/distutils/__init__.py:4: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/pkg_resources/_vendor/pyparsing.py:943: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  collections.MutableMapping.register(ParseResults)
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:26: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  def convert_to_list(value, n, name, dtype=np.int):
-----------  Configuration Arguments -----------
checkpoint_path: None
config: conf/deepspeech2.yaml
device: cpu
dump_config: None
export_path: None
nprocs: 4
opts: []
output: ckpt-baseline
------------------------------------------------
data:
  augmentation_config: conf/augmentation.config
  batch_size: 64
  dev_manifest: data/manifest_zh.dev
  keep_transcription_text: False
  max_duration: 27.0
  max_freq: None
  mean_std_filepath: data/mean_std_zh.npz
  min_duration: 0.0
  n_fft: None
  num_workers: 0
  random_seed: 0
  shuffle_method: batch_shuffle
  sortagrad: True
  specgram_type: linear
  stride_ms: 10.0
  target_dB: -20
  target_sample_rate: 16000
  test_manifest: data/manifest_zh.test
  train_manifest: data/manifest_zh.train
  use_dB_normalization: True
  vocab_filepath: data/vocab_zh.txt
  window_ms: 20.0
decoding:
  alpha: 2.5
  batch_size: 128
  beam_size: 300
  beta: 5.0
  cutoff_prob: 0.99
  cutoff_top_n: 40
  decoding_method: ctc_beam_search
  error_rate_type: cer
  lang_model_path: data/lm/zh_giga.no_cna_cmn.prune01244.klm
  num_proc_bsearch: 10
model:
  num_conv_layers: 2
  num_rnn_layers: 3
  rnn_layer_size: 2048
  share_rnn_weights: False
  use_gru: True
training:
  global_grad_clip: 5.0
  lr: 0.002
  lr_decay: 0.83
  n_epoch: 5
  weight_decay: 1e-06
[INFO 2021/04/21 02:47:33 model.py:212] Setup train/valid Dataloader!
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_in.conv.weight: shape: [32, 1, 41, 11] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_in.bn.weight: shape: [32] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_in.bn.bias: shape: [32] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_in.bn._mean: shape: [32] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_in.bn._variance: shape: [32] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_stack.0.conv.weight: shape: [32, 32, 21, 11] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_stack.0.bn.weight: shape: [32] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_stack.0.bn.bias: shape: [32] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_stack.0.bn._mean: shape: [32] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.conv.conv_stack.0.bn._variance: shape: [32] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_fc.weight: shape: [1312, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_fc.weight: shape: [1312, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.fw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.0.bw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_fc.weight: shape: [4096, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_fc.weight: shape: [4096, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.fw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.1.bw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_fc.weight: shape: [4096, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_fc.weight: shape: [4096, 6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_bn.weight: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_bn.bias: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_bn._mean: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_bn._variance: shape: [6144] stop_grad: True
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.fw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_cell.weight_hh: shape: [6144, 2048] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: encoder.rnn.rnn_stacks.2.bw_cell.bias_hh: shape: [6144] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: decoder.ctc_lo.weight: shape: [4096, 39] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:76] param: decoder.ctc_lo.bias: shape: [39] stop_grad: False
[INFO 2021/04/21 02:47:35 layer_tools.py:78] Total parameters: 192877959.0!
Epoch 0: ExponentialDecay set learning rate to 0.002.
[INFO 2021/04/21 02:47:35 model.py:138] Setup model/optimizer/lr_scheduler!
[INFO 2021/04/21 02:47:35 trainer.py:165] Train Total Examples: 4
2021-04-21 02:47:35,603 - INFO - dataset sortagrad! epoch 0
[INFO 2021/04/21 02:47:35 model.py:82] Valid Total Examples: 4
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/layers/utils.py:77: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
  return (isinstance(seq, collections.Sequence) and
2021-04-21 02:47:45,505 - INFO - warpctc loss: Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
       [84.97895813 , 389.37243652, 471.27011108, 553.85168457])/[4] 
2021-04-21 02:47:45,505 - INFO - ctc loss: Tensor(shape=[1], dtype=float32, place=CPUPlace, stop_gradient=True,
       [1499.47314453])
/data/DeepSpeech/tools/venv/lib/python3.8/site-packages/paddle/fluid/dygraph/math_op_patch.py:236: UserWarning: The dtype of left and right variables are not the same, left dtype is VarType.FP32, but right dtype is VarType.INT32, the right dtype will convert to VarType.FP32
  warnings.warn(
[INFO 2021/04/21 02:47:45 model.py:100] Valid: Rank: 0, epoch: 1, step: 0, val_loss: 374.868286
2021-04-21 02:48:07,661 - INFO - [checkpoint] Saved model to ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:48:07,661 - INFO - [checkpoint] Saved optimzier state to ckpt-baseline/checkpoints/step-0.pdopt
Epoch 1: ExponentialDecay set learning rate to 0.00166.
2021-04-21 02:48:07,661 - INFO - dataset shuffle! epoch 1
[INFO 2021/04/21 02:48:07 model.py:82] Valid Total Examples: 4
2021-04-21 02:48:17,486 - INFO - warpctc loss: Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
       [84.97895813 , 389.37243652, 471.27011108, 553.85168457])/[4] 
2021-04-21 02:48:17,486 - INFO - ctc loss: Tensor(shape=[1], dtype=float32, place=CPUPlace, stop_gradient=True,
       [1499.47314453])
[INFO 2021/04/21 02:48:17 model.py:100] Valid: Rank: 0, epoch: 2, step: 0, val_loss: 374.868286
2021-04-21 02:48:39,557 - INFO - [checkpoint] Saved model to ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:48:39,557 - INFO - [checkpoint] Saved optimzier state to ckpt-baseline/checkpoints/step-0.pdopt
Epoch 2: ExponentialDecay set learning rate to 0.0013778.
2021-04-21 02:48:39,558 - INFO - dataset shuffle! epoch 2
[INFO 2021/04/21 02:48:39 model.py:82] Valid Total Examples: 4
2021-04-21 02:48:49,357 - INFO - warpctc loss: Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
       [84.97895813 , 389.37243652, 471.27011108, 553.85168457])/[4] 
2021-04-21 02:48:49,357 - INFO - ctc loss: Tensor(shape=[1], dtype=float32, place=CPUPlace, stop_gradient=True,
       [1499.47314453])
[INFO 2021/04/21 02:48:49 model.py:100] Valid: Rank: 0, epoch: 3, step: 0, val_loss: 374.868286
2021-04-21 02:49:00,367 - INFO - [checkpoint] Saved model to ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:49:00,367 - INFO - [checkpoint] Saved optimzier state to ckpt-baseline/checkpoints/step-0.pdopt
Epoch 3: ExponentialDecay set learning rate to 0.0011435739999999998.
2021-04-21 02:49:00,368 - INFO - dataset shuffle! epoch 3
[INFO 2021/04/21 02:49:00 model.py:82] Valid Total Examples: 4
2021-04-21 02:49:10,201 - INFO - warpctc loss: Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
       [84.97895813 , 389.37243652, 471.27011108, 553.85168457])/[4] 
2021-04-21 02:49:10,202 - INFO - ctc loss: Tensor(shape=[1], dtype=float32, place=CPUPlace, stop_gradient=True,
       [1499.47314453])
[INFO 2021/04/21 02:49:10 model.py:100] Valid: Rank: 0, epoch: 4, step: 0, val_loss: 374.868286
2021-04-21 02:49:32,300 - INFO - [checkpoint] Saved model to ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:49:32,300 - INFO - [checkpoint] Saved optimzier state to ckpt-baseline/checkpoints/step-0.pdopt
Epoch 4: ExponentialDecay set learning rate to 0.0009491664199999998.
2021-04-21 02:49:32,301 - INFO - dataset shuffle! epoch 4
[INFO 2021/04/21 02:49:32 model.py:82] Valid Total Examples: 4
2021-04-21 02:49:41,987 - INFO - warpctc loss: Tensor(shape=[4], dtype=float32, place=CPUPlace, stop_gradient=True,
       [84.97895813 , 389.37243652, 471.27011108, 553.85168457])/[4] 
2021-04-21 02:49:41,988 - INFO - ctc loss: Tensor(shape=[1], dtype=float32, place=CPUPlace, stop_gradient=True,
       [1499.47314453])
[INFO 2021/04/21 02:49:41 model.py:100] Valid: Rank: 0, epoch: 5, step: 0, val_loss: 374.868286
2021-04-21 02:50:04,095 - INFO - [checkpoint] Saved model to ckpt-baseline/checkpoints/step-0.pdparams
2021-04-21 02:50:04,095 - INFO - [checkpoint] Saved optimzier state to ckpt-baseline/checkpoints/step-0.pdopt
Epoch 5: ExponentialDecay set learning rate to 0.0007878081285999999.

**Comment by zh794390558** [comment]
这个数据集太小了， val_loss: 374.868286， epoch 5 。这根本没法训练ASR。
